actix/src/sec-7-stream.md

```diff
@@ -1 +1,305 @@
-[WIP] 
\ No newline at end of file
+# Streams
+
+Actix allows the handling of Tokio Streams to act as a source of events
+or messages to actors. This is commonly used with networking servers
```

Comment:
>This is commonly used with networking servers

Not necessary?
Rather than saying doubtful statement like that, you should mention benefits of `Stream` as abstraction.
I.e. it provides you with future's compatible interface for incoming messages

Reply:
I don't understand how this can be used as an abstraction, and to me the value is in it being used for networking?

Reply:
`Stream` is abstraction that represents async items that will be available in future.
That's its usage which is possible only when your networking protocol allows framing your data(i.e. converting arbitrary bytes into complete message(frame))
Not all protocols allow such and therefore `Stream` is not necessary common for use